### DISCUSSION TITLE
Is there a way to put this behind a base path?

### DISCUSSION THREAD

**Discussion by MajliTech** [blakeblackshear/frigate#7669] [Ask A Question]
Basically the title. I would like to put it under /cctv instead of /, but the docs do not specify how.

**Comment by NickM-27** [comment]
Put what behind a base path?

**Reply by MajliTech** [reply]
This project

**Reply by NickM-27** [reply]
You could be talking about storage, you could be talking about the webUI, api, etc. if you want an answer you need to provide more info.

**Reply by MajliTech** [reply]
ui, api and live. i just want to access this on myserv.er/cctv/cameras/(whatever)

**Reply by NickM-27** [reply]
That would be something to configure in the web server that is handling the domain, or behind a reverse proxy like nginx, traefik, etc.

**Reply by MajliTech** [reply]
I put it behind, its just the config, when i access it i get a white site.
